components/extensions/org.wso2.carbon.ml.siddhi.extension/src/main/java/org/wso2/carbon/ml/siddhi/extension/PredictStreamProcessor.java

```diff
@@ -132,23 +202,24 @@ public void start() {
         try {
             populateFeatureAttributeMapping();
         } catch (Exception e) {
```

Comment:
Always better to put a comment if there's a valid reason we're catching a generic Exception.